conda-forge.yml

```diff
@@ -1,8 +1,11 @@
-os_version:
-  linux_64: cos7
+build_platform: 
+  linux_aarch64: linux_64
 conda_forge_output_validation: true
 github:
   branch_name: main
   tooling_branch_name: main
 conda_build:
   pkg_format: '2'
+provider:
+  linux_aarch64: default
```

Comment:
```suggestion
  linux_aarch64: default
  linux_ppc64le: default
```

Reply:
have yo utried? if not, lets try it?

Reply:
sure! should we add also to `build_platform`?

Reply:
oh, looks like the `build_platform` is automatically populated if you specify the `provider`?